recipes/jalview/meta.yaml

```diff
@@ -29,8 +29,9 @@ test:
 
 
 about:
-  home: http://www.jalview.org/
+  home: https://www.jalview.org/
   license: GPL3
+  license_file: LICENSE
```

Comment:
The Jalview tarball comes with a LICENSE file, so the copy added to the repository here can be removed.